src/routes/facility-types.js

```diff
@@ -0,0 +1,15 @@
+import {Router} from 'express'
+import utils from '../utils'
+
+const router = new Router()
+router.use(utils.authMiddleware)
+
+router.get('/', async (req, res) => {
+    return res.status(200).send([
```

Comment:
I think we need to make this dynamic, potentially reading from the DB. Our goal is to keep the back end generic and not-specific to HealthcareRoll.